src/store/state/state-model.ts

```diff
@@ -103,5 +103,13 @@ export const stateModel: StateModelIF = {
         agreementType: ''
       }
     }
+  },
+  staffPayment: {
+    option: NaN,
+    routingSlipNumber: '',
```

Comment:
As per my previous comment, would it better to _not_ assign some of these properties?
(And not set priority to false?)

Reply:
Never mind. Answered in another conversation.